st2api/tests/unit/controllers/v1/test_actions.py

```diff
@@ -854,3 +856,12 @@ def __do_delete(self, action_id, expect_errors=False):
         return self.app.delete(
             "/v1/actions/%s" % action_id, expect_errors=expect_errors
         )
+
```

Comment:
- Update existing unit test `test_delete` where there's no payload and assert database record is removed and mocked delete_action_files_from_pack is not called
- Add new unit test `test_delete_remove_files_false` where remove_files=False in payload and assert database record is removed and mocked delete_action_files_from_pack is not called
- Add new unit test `test_delete_remove_files_true` where remove_files=True in payload and assert database record is removed and mocked delete_action_files_from_pack is called